### PR TITLE
Useless text files

### PR DESCRIPTION
**Pull request by jschueller**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
@conda-forge-admin, please rerender

File: recipe/bld.bat [hunk 1]
```diff
@@ -32,3 +32,5 @@ if errorlevel 1 exit 1
 :: Install.
 nmake install
 if errorlevel 1 exit 1
+
+del %LIBRARY_PREFIX%\*.txt %LIBRARY_PREFIX%\COPYING
```
Comment (ocefpaf) [inline recipe/bld.bat:32]: I would leave the `COPYING` file. Even though conda has that in the package metadata some users do look for it in the "software designated folder."

**Pull request closed without merging**